### PR TITLE
SITL Params for GPS_NUMSATS and GPS_BYTELOSS

### PR DESCRIPTION
**Pull request by antholuo**
No output yet (will update when I get it up), very small PR just making sure I'm doing this right.

Not sure on when "USER: Advanced" is needed either.

### CODE REVIEW

File: libraries/SITL/SITL.cpp [hunk 1]
```diff
@@ -469,7 +469,14 @@ const AP_Param::GroupInfo SIM::var_gps[] = {
     // @Values: 0:None, 1:UBlox, 5:NMEA, 6:SBP, 7:File, 8:Nova, 9:SBP, 10:GSOF, 19:MSP
     // @User: Advanced
     AP_GROUPINFO("GPS_TYPE",       3, SIM,  gps_type[0],  GPS::Type::UBLOX),
+    // @Param: GPS_BYTELOSS
+    // @DisplayName: GPS Byteloss
+    // @Description: Percent of packages lost from GPS 1
```
Comment (peterbarker) [inline libraries/SITL/SITL.cpp:474]:
```suggestion
    // @Description: Percent of bytes lost from GPS 1
```

File: libraries/SITL/SITL.cpp [hunk 2]
```diff
@@ -493,7 +500,14 @@ const AP_Param::GroupInfo SIM::var_gps[] = {
     // @DisplayName: GPS 2 type
     // @Description: Sets the type of simulation used for GPS 2
     AP_GROUPINFO("GPS2_TYPE",     32, SIM,  gps_type[1],  GPS::Type::UBLOX),
+    // @Param: GPS_BYTELOSS
+    // @DisplayName: GPS Byteloss
+    // @Description: Percent of packages lost from GPS 2
```
Comment (peterbarker) [inline libraries/SITL/SITL.cpp:505]:
```suggestion
    // @Description: Percent of bytes lost from GPS 2
```

**Review by peterbarker — CHANGES_REQUESTED** [review]
This should be just two commits:
`SITL: document SIM_GPS_BYTELOS and SIM_GPS_NUMSATS` and
`Tools: document SIM_GPS_BYTELOS and SIM_GPS_NUMSATS` and

You can do this by (see https://ardupilot.org/dev/docs/git-interactive-rebase.html):
 - making a backup of your git directory
 - making sure everything is committed
 - git checkout master
 - git pull
 - git checkout ALUO_SITL_params
 - git rebase -i master
 - a text  editor should pop up
 - change "pick" on the first line to "r" in the editor
 - change "pick" on all subsequent lines to "f"
 - save the file and exit the editor
 - a text editor should pop up
 - change the commit message to "document SIM_GPS_BYTELOS and SIM_GPS_NUMSATS"
 - save the file and quit the editor
 - ./Tools/gittools/git-subsystems-split
 - git push -f YOURREMOTENAME

**Review by peterbarker — CHANGES_REQUESTED** [review]
@antholuo need to do the `./Tools/gittools/git-subsystems-split` step

**Review by peterbarker — APPROVED** [review]
LGTM

**Comment by antholuo** [comment]
Updated onto master & did an interactive rebase.

File: libraries/SITL/SITL.cpp [hunk 3]
```diff
@@ -493,7 +500,14 @@ const AP_Param::GroupInfo SIM::var_gps[] = {
     // @DisplayName: GPS 2 type
     // @Description: Sets the type of simulation used for GPS 2
     AP_GROUPINFO("GPS2_TYPE",     32, SIM,  gps_type[1],  GPS::Type::UBLOX),
+    // @Param: GPS_BYTELOSS
+    // @DisplayName: GPS Byteloss
+    // @Description: Percent of bytes lost from GPS 2
+    // @User: Advanced
     AP_GROUPINFO("GPS2_BYTELOS",  33, SIM,  gps_byteloss[1],  0),
```
[the remark this answers is not in the document]
Comment (peterbarker) [reply libraries/SITL/SITL.cpp:507]: > Noticed that for GPS2 it is `GPS2_BYTELOS` while for GPS1 it is `GPS_BYTELOSS`.
> 
> wanted to confirm this is intended?

parameter-name-length limit :-(

File: libraries/SITL/SITL.cpp [hunk 4]
```diff
@@ -493,7 +500,14 @@ const AP_Param::GroupInfo SIM::var_gps[] = {
     // @DisplayName: GPS 2 type
     // @Description: Sets the type of simulation used for GPS 2
     AP_GROUPINFO("GPS2_TYPE",     32, SIM,  gps_type[1],  GPS::Type::UBLOX),
+    // @Param: GPS_BYTELOSS
```
Comment (peterbarker) [inline libraries/SITL/SITL.cpp:503]:
```suggestion
    // @Param: GPS_BYTELOS
```

**Review by peterbarker — CHANGES_REQUESTED** [review]

**Review by peterbarker — APPROVED** [review]

**Comment by peterbarker** [comment]
Merged, thanks!

**Pull request merged**